### PR TITLE
Update collaborationInvite.html

### PR DESCRIPTION
**Pull request by goddesswarship**
**Changes**
- Fixed code formatting
- Removed references to whale sharks and whaleshark.org with giraffes and GiraffeSpotter
- Updated copyright
- Replaced Wild Me company references with Conservation X Labs

### CODE REVIEW

**Comment by goddesswarship** [comment]
@holmbergius Changes made ahead of meeting to fix general GiraffeSpotter bugs.

**Review by holmbergius — CHANGES_REQUESTED** [review]
Changes requested:

- PNG white text in header for clean readability
- Facebook link to Wild Me
- X link to Wild Me
- Link to donate page is currently just to giraffespotter

File: src/main/resources/emails/en/collaborationInvite.html [hunk 1]
```diff
@@ -1 +1,322 @@
...
+                <table class="row collapse" style="border-collapse:collapse;border-spacing:0;display:table;padding:0;position:relative;text-align:left;vertical-align:top;width:100%">
+                  <tbody>
+                    <tr style="padding:0;text-align:left;vertical-align:top"><th class="small-6 large-6 columns first" style="Margin:0 auto;color:#0a0a0a;font-family:Helvetica,Arial,sans-serif;font-size:16px;font-weight:400;line-height:1.3;margin:0 auto;padding:0;padding-bottom:0;padding-left:0;padding-right:0;text-align:left;width:298px">
+                      <table style="border-collapse:collapse;border-spacing:0;padding:0;text-align:left;vertical-align:top;width:100%">
+                        <tr style="padding:0;text-align:left;vertical-align:top">
+                          <th style="Margin:0;color:#0a0a0a;font-family:Helvetica,Arial,sans-serif;font-size:16px;font-weight:400;line-height:1.3;margin:0;padding:0;text-align:left">
+                            <img src="https://giraffespotter.org/cust/mantamatcher/img/GS_Spotter07302017_scaled.png" style="-ms-interpolation-mode:bicubic;clear:both;display:block;max-width:100%;outline:0;text-decoration:none;width:auto">
```
Comment (goddesswarship) [inline src/main/resources/emails/en/collaborationInvite.html:31]: @holmbergius There's only this and the non-scaled version of this image for the available GiraffeSpotter logos that I've found in the repository so far. Do I need to upload a different non-transparent image to replace it?

Comment (goddesswarship) [reply src/main/resources/emails/en/collaborationInvite.html]: Nevermind; the issue was with the background color, not the logo image.

**Comment by goddesswarship** [comment]
Ok, commit 0f2b770b003d484c07990ef5bce3fcacc2fb530d is ready for review.

**Review by holmbergius — APPROVED** [review]
HTML displays cleanly.

**Pull request merged**